docs/source/lightly.api.rst

```diff
@@ -18,14 +18,5 @@ lightly.api
 .. automodule:: lightly.api.api_workflow_selection
    :members:
 
-.. automodule:: lightly.api.api_workflow_upload_dataset
-   :members:
-
-.. automodule:: lightly.api.api_workflow_upload_embeddings
-   :members:
-
-.. automodule:: lightly.api.api_workflow_upload_metadata
-   :members:
-
```

Comment:
@guarin 
I think that removing these makes sense, as we don't want customers to use the upload via pip directly anymore.

Reply:
`api_workflow_upload_embeddings ` and `api_workflow_upload_metadata ` also contain methods to download things which we should still expose.

Reply:
I did not find any methods that users should use in those files. Which ones do you mean?
The download of embeddings is inside `api_workflow_download_dataset.py`.

Reply:
Ah sorry, I though `verify_custom_metadata_format` was needed but it isn't, so `api_worklfow_upload_metadata` can be removed. From `api_workflow_upload_embeddings` I would have kept `get_embedding_by_name` or do we not support exporting emebeddings?

Reply:
`get_embedding_by_name` looks like a duplicate of [api_workflow_download_dataset.py:get_embedding_data_by_name](https://github.com/lightly-ai/lightly/blob/2a0f7e8edff98634bdd3bfd44df748565ace6883/lightly/api/api_workflow_download_dataset.py#L152)

Reply:
Ah nice, then we can remove it.